NOTICE

```diff
@@ -0,0 +1,6 @@
+CycloneDX Editor Validator
+Copyright (c) Festo SE & Co. KG
```

Comment:
This is a policy for Apache Software Foundation projects. It is not a requirement for NOTICE files of projects licensed under the Apache license, much less projects like ours licensed under a different license which only include a NOTICE file because one of their dependencies requires it.

If you like, we can still add the year. It's just a small burden to maintain, as you say.

Reply:
Is not there an action for that? Or maybe we need an own action, which is triggered on the 1st January, updating the year in the `NOTICE`, `LICENSE` and `mkdocs.yml`?